doc/manual/source/start/installation.rst

```diff
@@ -41,9 +41,9 @@ To build GeoGig and have an executable binary that you can run:
 
    .. note:: To speed up the build process, you can skip tests:
 
-             ::
+   ::
```

Comment:
The indent here, and the one below were intended to give a literal block inside a note appearance. Removing the indentation on the literal block `::` marker and the next line that starts `mvn clean install` moves the Maven command line out as a separate literal block that looks less connected to the Note.